provision/docker_test.go

```diff
@@ -338,34 +338,3 @@ func TestFnKilContainerNotFound(t *testing.T) {
 		t.Errorf("expecting errors, but nothing found")
 	}
 }
-
-func TestFnConfigVolumeAllEmpty(t *testing.T) {
-	volume := FnConfigVolume(&VolumeOptions{})
-	if volume != "" {
-		t.Errorf("Expected \"\" but found %q", volume)
-	}
-}
-
-func TestFnConfigVolumeDestinationOmitted(t *testing.T) {
-	volume := FnConfigVolume(&VolumeOptions{Source: "/tmp"})
-	if volume != "/tmp:/tmp" {
```

Comment:
We have tests using volume, see line 130 of provision/docker_test.go. These deleted tests are about the old volume structure.